packages/components/tests/dummy/app/templates/components/form/radio-card.hbs

```diff
@@ -0,0 +1,625 @@
+    is an input of type "radio" with a bigger target area represented by a card container. Same as the
+    <LinkTo @route="components.form.radio"><code class="code-dummy">Form::Radio</code></LinkTo>
+    component, the
+    <code class="code-dummy">RadioCard</code>
+    ​​ is a form element that allows users to select a single item from a list of related options.</p>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="component-api"><a href="#component-api" class="dummy-link-section">§</a> Component API</h3>
+  <h4 class="dummy-h4">Form::RadioCard</h4>
+  <p class="dummy-paragraph" id="component-api-form-radio-card">
+    Here is the API for the
+    <code class="code-dummy">RadioCard</code>
+    component:
+  </p>
+  <dl class="dummy-component-props" aria-labelledby="component-api-form-radio-card">
+    <dt>name <code>string</code></dt>
+    <dd>
+      <p>The input control's name attribute</p>
+    </dd>
+    <dt>value <code>string</code></dt>
+    <dd>
+      <p>The input control's value attribute</p>
+    </dd>
+    <dt>checked <code>boolean</code></dt>
+    <dd>
+      <p>The input control's checked attribute</p>
+    </dd>
+    <dt>disabled <code>boolean</code></dt>
+    <dd>
+      <p>The input control's disabled attribute</p>
+    </dd>
+    <dt>extraAriaDescribedBy <code>string</code></dt>
+    <dd>
+      <p>An extra ID attribute to be added to the <code class="dummy-code">aria-describedby</code> HTML attribute.</p>
+      <p><em>Notice: by default the
+          <code class="dummy-code">aria-describedby</code>
+          attribute is automatically generated by the component, using the IDs of the helper text and errors (if they're
+          present); use this argument if you need to pass an extra ID for specific reasons you may have.</em></p>
+    </dd>
+    <dt>...attributes</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+      <p><em>Notice: the attributes will be applied to the
+          <code class="dummy-code">&lt;input type="radio"&gt;</code>
+          element. This means you can use all the standard HTML attributes of the
+          <code class="dummy-code">&lt;input type="radio"&gt;</code>
+          element and all the usual Ember techniques for event handling, validation, etc.</em></p>
+      <p><em>Some examples of HTML attributes that you will likely use:
+          <code class="dummy-code">id</code>,
+          <code class="dummy-code">disabled</code>
+          (<a
+            href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attributes"
+            rel="noopener noreferrer"
+          >see whole list here</a>) and some examples of Ember modifiers:
+          <code class="dummy-code">&lcub;&lcub;on "click" [do something]&rcub;&rcub;</code>,
+          <code class="dummy-code">&lcub;&lcub;on "change" [do something]&rcub;&rcub;</code>.
+        </em></p>
+    </dd>
+  </dl>
+  <h5 class="dummy-h5">Contextual components</h5>
```

Comment:
as mentioned above, most of these contextual sub-component accept attributes spreading, so maybe we should make it explicit in the documentation